minimal_counter_dapp/src/declarations/minimal_counter_dapp/index.js

```diff
@@ -0,0 +1,38 @@
+import { Actor, HttpAgent } from "@dfinity/agent";
```

Comment:
I'm going to verify, but I think `src/declarations` should be in .gitignore, not committed to source control.

Reply:
I see your point.